.github/actions/setup-buildx-cache/action.yml

```
@@ -0,0 +1,19 @@
+name: Setup Docker Buildx & Cache Docker layers
+description: a composite action to simplify other workflows
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@master
+      with:
+        install: true
+
+    - name: Cache Docker layers
+      uses: actions/cache@v2
+      with:
+        path: /tmp/.buildx-cache-${{ github.sha }}
+        # Key is named differently to avoid collision
+        key: ${{ runner.os }}-multi-buildx-${{ github.sha }}
+        restore-keys: |
+          ${{ runner.os }}-multi-buildx
```

```suggestion
          ${{ runner.os }}-multi-buildx
          
```

usually it is advisable to end a file with a line return

---

Maybe adding more detailed description to the composite action as suggested by @gonuke would be nice.